src/blurredLocation.js

```diff
@@ -281,10 +281,24 @@ BlurredLocation = function BlurredLocation(options) {
     updateRectangleOnPan();
 
   function geocodeWithBrowser(boolean) {
-    if ("geolocation" in navigator) {
-      navigator.geolocation.getCurrentPosition(function(position) {
-      goTo(position.coords.latitude, position.coords.longitude,options.zoom);
-      });
+    if(boolean) {
+      var label = document.createElement("label");
+      label.classList.add("spinner");
+      var i = document.createElement("i");
+      i.classList.add("fa");
+      i.classList.add("fa-spinner");
+      i.classList.add("fa-spin");
+      label.appendChild(i);
+      var element = document.getElementById("div1");
```

Comment:
Hmm, should this be dynamically set, though? I think we perhaps should default to something like `ldi-geocode-button` -- what do you think?

Reply:
If we keep it static how do we render the spinner? Also changing the id

Reply:
We can say:

```js
var geocodeButtonId = geocodeButtonId || "ldi-geocode-button";
var element = document.getElementById(geocodeButtonId);`
```

And add to the demo and documentation that you can either make a button with `id=ldi-geocode-button` OR override it by setting `geocodeButtonId` in the options. How's that? 

Thank you!!!!